frontend/src/constants/content-safety.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Contains constants pertaining to the content safety feature. These include
+ * different types of sensitivities.
+ */
+
+export const USER_REPORTED = "user_reported_sensitive"
+export const PROVIDER_SUPPLIED = "provider_supplied_sensitive"
+export const TEXT_FILTERED = "sensitive_text"
+
+export const SENSITIVITIES = [
+  USER_REPORTED,
+  PROVIDER_SUPPLIED,
+  TEXT_FILTERED,
+] as const
+
+export type Sensitivity = (typeof SENSITIVITIES)[number]
```

Comment:
Just a question: are then parens here necessary? It shouldn't be, as far as I can tell (and from experience): https://www.typescriptlang.org/play?#code/MYewdgzgLgBAggJQTAvDA2gIgIaYDQyYBG+hwmAujNhDKJFAFCNQCeADgKbwBOPqMNlxAAzeEnRgArgFsinHhSA

Is something else going on here that makes them required, by chance?

Reply:
The only reason I know is that Prettier adds them if you don't.